Dart/resources/META-INF/plugin.xml

```diff
@@ -153,6 +153,8 @@
     <lang.findUsagesProvider language="Dart" implementationClass="com.jetbrains.lang.dart.ide.findUsages.DartFindUsagesProvider"/>
     <usageTypeProvider implementation="com.jetbrains.lang.dart.ide.findUsages.DartUsageTypeProvider"/>
     <findUsagesHandlerFactory implementation="com.jetbrains.lang.dart.ide.findUsages.DartServerFindUsagesHandlerFactory"/>
+    <fileStructureGroupRuleProvider implementation="com.jetbrains.lang.dart.ide.findUsages.UnitMemberGroupRuleProvider"/>
+    <fileStructureGroupRuleProvider implementation="com.jetbrains.lang.dart.ide.findUsages.ClassMemberGroupRuleProvider"/>
```

Comment:
We have 25 group rule providers for different languages in our code base, among them 8 are called `*Class*GroupRuleProvider`. So to make developer's life easier I suggest to call all Dart specific classes starting with 'Dart'. DartUnitMemberGroupRuleProvider, DartClassMemberGroupRuleProvider. The rule is more important for extensions and classes that extend Platform classes. But I prefer to have Dart word always, it's much easier to guess the class name this way when you are looking for something.